app/controllers/student_requests_controller.rb

```diff
@@ -473,11 +475,13 @@ def multiupdate
             isUpdate = true
             @student_request.state = session[:multi_state_sel]
               @student_request.save!
-              message = params[:email_message][0]
+              
             admin_log(@student_request.request_id, "Status of the request was updated to #{@student_request.state} by #{session[:uin]}")
             if(@student_request.state != StudentRequest::HOLD_STATE)
+              message = params[:email_message][0]
```

Comment:
This is not up to date with the master branch. the code is different in the master branch. this will create merge conflict. resolve this conflict by updating your branch to the master branch code.